### PR TITLE
Feature interactive list

### PR DESCRIPTION
**Pull request by emmyobonyo**
- A User can cross-out a completed task but clicking on the checkbox.
- A user can delete all completed tasks using the clear button

### CODE REVIEW

File: src/render.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  },
+  {
+    index: 3,
+    isCompleted: false,
+    description: 'Take breakfast',
+  },
+];
+
+localStorage.setItem('tasks', JSON.stringify(tasks));
+const getTasks = JSON.parse(localStorage.getItem('tasks'));
```
Comment (vichuge) [inline src/render.js:28]: - [ ] If I check a task as completed and then I refresh the page, the local storage doesn't save the new status. Please, add the functionality to make this. 🐶 
- hint: I check all your script but I'm afraid I don't find a function who save the new status.

File: src/render.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
+const ul = document.getElementById('list-items');
```
Comment (vichuge) [inline src/render.js:1]: - [ ] In the entire script I couldn't find an event listener who saves the new status for every checkbox element. Please, add one event listener and the function to change the element status. 🐱 
- I check my revision, I'm in the correct project with the correct branch, installing everything locally.
image

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by emmyobonyo** [comment]
- Checked Items remain checked and crossed out on refresh.
- Cleared items remain cleared on refresh.

**Review by hemant-soni-vst-au4 — APPROVED** [review]
Hi @emmyobonyo, 

Good work on making the requested changes. 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Alt Text

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**